examples/testerina-assertions/testerina_assertions.md

```diff
@@ -1,11 +1,12 @@
 # Assertions
 
-Testerina has in-built assertions that enable you to
-assert an outcome against an expected outcome.
-This example illustrates how to use different assertions.<br/><br/>
-For more information, see [Testing Ballerina Code](https://ballerina.io/learn/testing-ballerina-code/testing-quick-start/)
-and the [Test Module](https://docs.central.ballerina.io/ballerina/test/latest/).
+Testerina has in-built assertions that enable you to assert an outcome against
+an expected outcome. This example illustrates how to use different assertions.
+
+For more information, see [Testing Ballerina Code](https://ballerina.
```

Comment:
```suggestion
For more information, see [Testing Ballerina Code](https://ballerina.
```
We have removed the `-ing` form in all Learn page titles and updated them to sentence case. Please check and fix all the occurrences.